DataProviderService/StarSystemRepository.cs

```diff
@@ -6,14 +6,14 @@ namespace EddiDataProviderService
     public interface StarSystemRepository
     {
         StarSystem GetStarSystem(string name, bool refreshIfOutdated = true);
-        StarSystem GetOrCreateStarSystem(string name, bool refreshIfOutdated = true);
-        StarSystem GetOrFetchStarSystem(string name, bool fetchIfMissing = true);
+        StarSystem GetOrCreateStarSystem(string name, bool fetchIfMissing = true, bool refreshIfOutdated = true);
```

Comment:
Since the first bool was `refreshIfOutdated` and is `fetchIfMissing`, this may change the behavior of functions calling this method. Has this been considered?

Reply:
The problem was that in the implementations, fetchIfMissing and refreshIfOutdated metaphors were mixed. For example, in GetOrCreateStarSystems, was used fetchIfMissing for both getting systems not currently in the database and refreshing stale data. I separated them out. Now, if you set both to `False`, only local database will be used.